resources/views/layouts/librenmsv1.blade.php

```diff
@@ -26,7 +26,7 @@
     <link href="{{ asset('css/bootstrap-datetimepicker.min.css') }}" rel="stylesheet" type="text/css" />
     <link href="{{ asset('css/bootstrap-switch.min.css') }}" rel="stylesheet" type="text/css" />
     <link href="{{ asset('css/toastr.min.css') }}" rel="stylesheet" type="text/css" />
-    <link href="{{ asset('css/jquery-ui.min.css') }}" rel="stylesheet" type="text/css" />
+{{--    <link href="{{ asset('css/jquery-ui.min.css') }}" rel="stylesheet" type="text/css" />--}}
```

Comment:
remove?

Reply:
I am unsure if jquery-ui is used anywhere and I can't figure out a good way to tell... Any suggestions are welcome.   Upgraded versions are hard to pull in because it is a modular build and requires us to know what we are using.